custom_components/alexa_media/config_flow.py

```diff
@@ -908,6 +911,7 @@ async def async_step_init(
                 user_input[CONF_OTPSECRET] = self.config_entry.data[CONF_OTPSECRET]
             if CONF_OAUTH in self.config_entry.data:
                 user_input[CONF_OAUTH] = self.config_entry.data[CONF_OAUTH]
+
```

Comment:
Not sure why this space was added but if it helps with your understanding it's fine.